packages/dd-trace/src/config-helper.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
    * @returns {string|undefined}
    * @throws {Error} if the configuration is not supported
    */
+  // This method, and callers of this method, need to be updated to check for declarative config sources as well.
   getEnvironmentVariable (name) {
```

Comment:
getEnvironmentVariable is called 150+ times across the codebase. I'm thinking the changes described on line 81 should be done in a separate PR.